src/openms/include/OpenMS/FORMAT/PercolatorInfile.h

```diff
@@ -42,7 +42,8 @@ namespace OpenMS
         const String& score_name, 
         const StringList& extra_scores,
         StringList& filenames, 
-        String decoy_prefix = "");
+        String decoy_prefix = "",
+        double threshhold = 0.01);
```

Comment:
```suggestion
        double threshold = 0.01);
```

Reply:
todo: add threshold to documentation above

Reply:
Do you mean in the Changelog documentation as a feature or above the definition of the method?

Reply:
declaration of method